### DISCUSSION TITLE
Upgrading Microsoft Centre of Excellence Error

### DISCUSSION THREAD

**Discussion by Blues1981** [community/community#57604] [New to GitHub]
Hi I am attempting to upgrade the MS COE but receiving this error any help greatly appreciated.

![Error D365](https://github.com/community/community/assets/133023969/6a494a83-22ab-4385-b84c-9d60f793f55a)

**Comment by queenofcorgis** [comment]
Hi @Blues1981 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Center of Excellence like https://powerusers.microsoft.com/. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Blues1981** [reply]
Ok thank you
